assets/queries/terraform/alicloud/rds_instance_ssl_action_not_open/query.rego

```diff
@@ -0,0 +1,36 @@
+package Cx
+
+import data.generic.common as common_lib
+
+incorrect_values := {"Close","Update"}
+
+CxPolicy[result] {
+	some i
+	resource := input.document[i].resource.alicloud_db_instance[name]
+	resource.ssl_action == incorrect_values[val]
```

Comment:
I think we can consider a "vulnerability" when is only set to `Close`. Let me know what you think 😊